Project.toml

```diff
@@ -4,10 +4,16 @@ authors = ["Youngdae Kim <youngdae@anl.gov>"]
 version = "0.1.0"
 
 [deps]
+AMDGPU = "21141c5a-9bdb-4563-92ae-f87d6854732e"
```

Comment:
I guess here we have the same question as usual: do we want to be dependent both from CUDA and AMDGPU? :-)